### PR TITLE
Mrc-3275 - Bug: prevent spamming when creating a new project during model calibrate

### PR DESCRIPTION
**Pull request by LekanAnni**
## Description

This PR fixes issue continuous polling for results when polling in progress while a new project is created. I added commits to stop polling for modelRun and modelCalibrate if they exists. This method has stopped the known issue.

Please describe your changes here

## Type of version change
_Delete as appropriate_

Minor 

## Checklist

- [x] I have incremented version number, or version needs no increment
- [x] The build passed successfully, or failed because of ADR tests

### CODE REVIEW

File: src/app/static/src/app/store/modelCalibrate/mutations.ts [hunk 1]
```diff
@@ -138,6 +139,10 @@ export const mutations: MutationTree<ModelCalibrateState> = {
 
     [ModelCalibrateMutation.CalibrateResultFetched](state: ModelCalibrateState, action: PayloadWithType<CalibrateResultResponse>) {
         state.result = action.payload
+    },
+
+    [ModelCalibrateMutation.ResetIds](state: ModelCalibrateState) {
```
Comment (EmmaLRussell) [inline src/app/static/src/app/store/modelCalibrate/mutations.ts:144]: `ResetIds` seems a bit ambiguous (we have lots of ids!) - how about just `StopPolling` or `StopStatusPolling`?

File: src/app/static/src/app/store/modelRun/mutations.ts [hunk 2]
```diff
@@ -88,6 +89,10 @@ export const mutations: MutationTree<ModelRunState> = {
     [ModelRunMutation.ClearWarnings](state: ModelRunState) {
         state.warnings = [];
     },
+
+    [ModelRunMutation.ResetIds](state: ModelRunState) {
```
Comment (EmmaLRussell) [inline src/app/static/src/app/store/modelRun/mutations.ts:93]: ...similarly here

**Review by EmmaLRussell — APPROVED** [review]
Works great 👍 I guess this is less necessary for ModelRun, since running happens with a modal shown, so you can't create a new project when it's running anyway - but it's good to be consistent, and we might change the UI some day. 

I think the name of the mutation could be clearer, but I've just noticed that the Download module has that mutation name too, so it's following precedent! Up to you whether you want to change that or not - if you do, it should be changed for the Downloads too..

**Comment by LekanAnni** [comment]
> Works great 👍 I guess this is less necessary for ModelRun, since running happens with a modal shown, so you can't create a new project when it's running anyway - but it's good to be consistent, and we might change the UI some day.
> 
> I think the name of the mutation could be clearer, but I've just noticed that the Download module has that mutation name too, so it's following precedent! Up to you whether you want to change that or not - if you do, it should be changed for the Downloads too..

Looks good to me, I don't mind it.

**Pull request merged**